pyproject.toml

```diff
@@ -0,0 +1,41 @@
+[project]
+name = "pebble"
+version = "0.1.0"
+description = ""
+readme = "README.md"
+requires-python = ">=3.9"
+dependencies = [
+	"Pillow",
+	"freetype-py",
+	"ply==3.4",
+	"pyusb==1.3.1",
+	"pyserial",
+	"sh==1.04",
+	"pypng",
+	"pexpect",
+	"cobs==1.0.0",
+	"svg.path",
+	"requests",
+	"GitPython==1.0.1",
+	"pyelftools",
+	"pycryptodome",
+	"mock",
+	"nose",
+	"boto",
+	"prompt-toolkit>=0.55",
+	"enum34",
+	"bitarray",
+	"pep8",
+	"polib",
+	"intelhex>=2.1,<3",
+	"protobuf",
+	"grpcio-tools",
+	"nanopb",
+	"certifi",
+	"libclang",
+	"packaging",
+	
+	"libpebble2>=0.0.16",
+	"pyftdi==0.56.0",
+	"pathlib",
+]
```

Comment:
All changes lgtm except this: I don't think the pebble repo should be considered as-is a Python project. These dependencies are from random scripts used by e.g. the build system. I'd keep a requirements.txt for now.